### PR TITLE
PR TEST

### PR DESCRIPTION
**Pull request by SAMPROO**
Deleted unecesarry using
Added comment
PR TEST

### CODE REVIEW

File: MorkoUnityProject/Assets/Scripts/LocalController.cs [hunk 1]
```diff
@@ -58,21 +63,29 @@ public AvatarPackage Update()
...
        
 				float step = speed * Time.deltaTime;
 				character.transform.rotation = Quaternion.RotateTowards(lookRotation, character.transform.rotation, step);
 			}
-			// Keep old rotation
-			else if (joystickRotateX == 0 && joystickRotateY == 0 && mouseDelta.x == 0 && mouseDelta.y == 0)
+			// Controller being used, right joystick not being used, look towards player forward
+			if (joystickRotateX == 0 && joystickRotateY == 0 && mouseDelta.x == 0 && mouseDelta.y == 0 && mouseControl == false)
+			{
+				Debug.Log("CONTROLLER TURE BUTON NOT ROTATING");
```
Comment (Iiropel) [inline MorkoUnityProject/Assets/Scripts/LocalController.cs]: I do no understand this debug log

**Review by Iiropel — CHANGES_REQUESTED** [review]
Testing testing

**Review by SAMPROO — COMMENTED** [review]
Changes should be ok
test test

**Review by Iiropel — APPROVED** [review]
yes

**Pull request merged**